### DISCUSSION TITLE
How can I get YOLO badge?

### DISCUSSION THREAD

**Discussion by tomapull** [EddieHubCommunity/support#6848] [Q&A]
I have many pull requests, but I don't have YOLO badge yet.
Please help me to gain that.

**Comment by Sayyidalijufri** [comment]
Follow the below steps.

1. Create a pull request on your own remote repository.
2. Open the pull request.
3. At the top right corner add a reviewer to your repository.
4. Before they review your code and merge you should merge your code to the remote repository.
